releases/release-1.5/release-team.md

```diff
@@ -6,8 +6,16 @@
 | Release Team Member(s) | Anna Jung (GitHub: [@annajung](https://github.com/annajung) / Slack: `@annajung`), Daniela Plascencia (GitHub: [@DnPlas](https://github.com/DnPlas) / Slack: `@DnPlas`), Dominik Fleischmann (GitHub: [@DomFleischmann](https://github.com/DomFleischmann) / Slack: `Dominik Fleischmann`), Kylie Travis (GitHub: [@Bhakti087](https://github.com/Bhakti087) / Slack: `Kylie Travis`), Mathew Wicks (GitHub: [@thesuperzapper](https://github.com/thesuperzapper) / Slack: `Mathew Wicks`), Suraj Kota (GitHub: [@surajkota](https://github.com/surajkota) / Slack: `Suraj Kota`), Vedant Padwal (GitHub: [@js-ts](https://github.com/js-ts) / Slack: `Vedant Padwal`)|
 | Product Manager | Josh Bottum (GitHub: [@jbottum](https://github.com/jbottum) / Slack: `@JoshBottum`) |
 | Docs Lead | Shannon Bradshaw (GitHub: [@shannonbradshaw](https://github.com/shannonbradshaw) / Slack: `@Shannon Bradshaw`) |
-| Working Group Liaison(s) | |
-| Distribution Representative |  |
+
+| **Working Group** | **Liaison(s)** (**GitHub / Slack ID**) |
```

Comment:
Just curious, how were the liaisons selected for this release?

Reply:
I took a quick look at the WG tracking issues of the previous release and included some of the people there. I only added one to not make this list too long, since we will be tagging these people a lot [aside from tagging the WG leads].

@terrytangyuan would you also like to be a liaison for Training?

Reply:
I see. I cannot commit anything this time around. The main reason why I asked is to make sure the people you've included are responsive, especially given the holidays.